src/lib/api.js

```diff
@@ -1,16 +1,15 @@
 import axios from 'axios';
 import config from '../config';
 
-const GOOGLE_GEOCODE_URL =
-  'https://maps.googleapis.com/maps/api/geocode/json?sensor=false&language=en';
```

Comment:
Would see this as remaining here instead of in config coz not something you'd change for a new instance.

Reply:
Someone in the config could add some query options @DavidLemayian

Reply:
@karimkawambwa I see the query options happening here where e.g we automatically pass TLD for the country biasing - https://developers.google.com/maps/documentation/geocoding/intro#RegionCodes

Also language I'd see as being dynamic to change according to user's preference of viewing the site.